packages/nuxt/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@posthog/nuxt",
+  "version": "0.0.1",
+  "description": "Nuxt module for Posthog 🦔",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PostHog/posthog-js.git",
+    "directory": "packages/nuxt"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/types.d.mts",
+      "import": "./dist/module.mjs"
+    }
+  },
+  "main": "./dist/module.mjs",
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/types.d.mts"
+      ]
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prepare": "nuxt prepare",
+    "build": "nuxt-module-build build",
+    "package": "pnpm pack --out $PACKAGE_DEST/%s.tgz"
```

would be cool to have a lint script even if it's not specific to nuxt

```suggestion
    "package": "pnpm pack --out $PACKAGE_DEST/%s.tgz",
    "lint": "eslint src"
```